ocs_ci/helpers/helpers.py

```diff
@@ -4776,3 +4776,22 @@ def is_rbd_default_storage_class(custom_sc=None):
 
     logger.error("Storageclass {default_rbd_sc} is not a default  RBD StorageClass.")
     return False
+def custom_round(number):
```

Comment:
I think, no new line before the end of the previous method return statement and  `custom_round` is one of the reasons PR does not pass PR validation (Tox) check

Reply:
done